docs/_appendix/github-intro.md

```diff
@@ -170,6 +170,26 @@ them to be reviewed and merged.
 
 7.  Repeat these steps as many times as you'd like before you submit.
 
+## Verify
+
+The Web Platform Test project has an automated tool
+to verify that coding conventions have been verified,
+and to catch a number of common mistakes.
+
+You can and should run this tool localy.
+It will be run on your submissions anyway,
+and any failure will be grounds for rejection,
+so you might as well fix them first.
+
+1. On the command line, navigate to to the directory where your forked copy of
+the repository is located.
```

Comment:
+1. On the command line, navigate to to the directory where your forked copy of

I think _clone_ would be better than _forked copy_ (and I think we use _clone_ in other places in the existing docs)